### PR TITLE
1.0 - Edition (Testing not sure if is all working right!)

### PR DESCRIPTION
**Pull request by C3-FrameWork**
Created and Edited by Its_Cehic.
Why I was making this?
Cause I want that normally every person which using ESX LEGACY using circle progress bar instead defaulted.

### CODE REVIEW

File: esx_circleprogressbar/__resource.lua [hunk 1]
```diff
@@ -0,0 +1,16 @@
+resource_manifest_version '44febabe-d386-4d18-afbe-5e627f4af937'
```
Comment (Cyzxin) [inline esx_circleprogressbar/__resource.lua:1]: Why __resource and not fxmanifest? resource is really old and shouldn't be used

**Review by Cyzxin — CHANGES_REQUESTED** [review]

File: esx_circleprogressbar/__resource.lua (hunk 1 above)
Comment (C3-FrameWork) [reply esx_circleprogressbar/__resource.lua:1]: I'm gonna update it!

**Comment by Cyzxin** [comment]
file is fxmanifest not fx_manifest

**Comment by C3-FrameWork** [comment]
an misunderstanding, now it is right!

File: esx_circleprogressbar/fxmanifest.lua [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+author 'Its_Cehic'
+version '1.0'
+description 'Circle Progress Bar - Test Version'
+-- Created and Edited by Its_Cehic
+
+client_script 'script.js'
+server_script 'progress.lua'
+
+ui_page 'index.html'
```
Comment (Cyzxin) [inline esx_circleprogressbar/fxmanifest.lua:12]: these need to be nui/filename not just filename due to file paths. and in the files {} beneath

**Comment by Cyzxin** [comment]
also, i can see a few things that may be issues. make sure this is tested and working, mabye send video clip?

**Review by Cyzxin — CHANGES_REQUESTED** [review]

File: esx_circleprogressbar/fxmanifest.lua (hunk 2 above)
Comment (C3-FrameWork) [reply esx_circleprogressbar/fxmanifest.lua]: I tested! Here is error about that is not defined the "document".
Here is the photo:
image

File: esx_circleprogressbar/nui/script.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (Cyzxin) [inline esx_circleprogressbar/nui/script.js:1]: that's this line here, what is document and why are you using it? Personally, you should get this working and submit a new PR!

**Review by Cyzxin — CHANGES_REQUESTED** [review]

**Comment by Gellipapa** [comment]
@C3-FrameWork Hi! This is completely wrong, the code is outdated, it is not used correctly. You have a script.js file and it's written in client_script when it has nothing to do with the client. Even if we do it, we'll do it in a way that supports the old one and most likely in some frontend framework so that we can extend it much better in the future and get even better code out of our hands.

**Comment by C3-FrameWork** [comment]
Okay

**Comment by Gellipapa** [comment]
NUI files, you need add files section in fxmanifest not client_script, don't use var in js file.. etc.

**Pull request closed without merging**